### PR TITLE
Added ourPerl as "ourperl" to `fetch`, added to default PATHs

### LINKED ISSUE #1364: integrating ourPerl into `fetch`

### PR DESCRIPTION
**Pull request by wwlwpd**
Issue 1364: Added our fork of Nate's Perl repo to the `fetch` command, use `fetch ourperl`. Updated default PATHs in asgs-brew to include the directories in git/ourPerl and git/asgs-mon; it should be okay that they are fixed like that, it's how all the others were added.

Resolves #1364.

### CODE REVIEW

File: bin/fetch [hunk 1]
```diff
@@ -16,75 +16,88 @@ case "${ITEM}" in
...
     git clone git@github.com:CERA-GROUP/cera-asgs-environment.git $SCRIPTDIR/git/cera-asgs-environment
 
-    popd > /dev/null 2>&1 
+    popd > /dev/null 2>&1
+    ;;
+  ourperl)
+    mkdir $SCRIPTDIR/git 2> /dev/null
+    pushd $SCRIPTDIR/git  > /dev/null  2>&1
+    if [ -d ./ourPerl ]; then
+      echo "${W} $SCRIPTDIR/git/asgs.wiki exists"
```
Comment (jasonfleming) [inline bin/fetch:73]: This looks like a cut-and-paste typo.

**Review by jasonfleming — COMMENTED** [review]
Thank you @wwlwpd ... is this ready for me to test?

File: bin/fetch (hunk 1 above)
Comment (wwlwpd) [reply bin/fetch:73]: Good catch! TY

**Comment by wwlwpd** [comment]
> Thank you @wwlwpd ... is this ready for me to test?

Yes, go for it. You will have to run this to update, though (outside of `asgsh`:

```
./init-asgs.sh -b -x "--update-shell"
```

The `asgsh` script needs to get regenerated so it has the update, fixed `PATH`.

**Comment by wwlwpd** [comment]
@jasonfleming - I fixed the copy n paste issue with `asgs.wiki` - gonna merge so I can get the other one in w/asgs-mon and `jq` update.

**Pull request merged**